### DISCUSSION TITLE
Feature request: Downloading the latest bs updates from app itself.

### DISCUSSION THREAD

**Discussion by pawarsanjeev** [efroemling/ballistica#736] [Ideas]
Can we add a feature in the app that notifies us when a new update for the game is available and allows the game to be updated directly from the app, or at least redirects the user to the official source? Thanks!

**Comment by Loup-Garou911XD** [comment]
there is this plugin by @brostosjoined which notifies you when there is an update
https://discord.com/channels/1001896771347304639/1376941773246431352
